### ISSUE TITLE
aax files download, somewhere (not in the download directory), but can't be assembled

### ISSUE THREAD

**Issue by philgoetz** [mkb79/audible-cli#69]
I can't download aax files at all.  audible-cli doesn't recognize them as aax files.
It looks like they're just tagged differently internally, as type 'audio/audible'.

```
$ audible download --all --aax  --cover --cover-size 1215 -j 1 --pdf
...
The_Problems_of_Philosophy-LC_64_22050_stereo.aax:  98%|█████████▊| 60.6M/62.1M
The_Problems_of_Philosophy-LC_64_22050_stereo.aax:  98%|█████████▊| 61.1M/62.1M
The_Problems_of_Philosophy-LC_64_22050_stereo.aax:  99%|█████████▉| 61.7M/62.1M
The_Problems_of_Philosophy-LC_64_22050_stereo.aax: 100%|██████████| 62.1M/62.1M [00:11<00:00, 5.47MB/s]
error: Error downloading /cygdrive/e/data/audio_/Audible/audible-cli/The_Problems_of_Philosophy-LC_64_22050_stereo.aax. Wrong content type. Expected type(s): ['audio/aax', 'audio/vnd.audible.aax']; Got: audio/audible; Message: Unknown
```

It looks like all the parts downloaded somewhere, but I don't see where.  No .aax files appear in the download directory.

**Comment by mkb79** [comment]
This issue is know and solved (see issue #62) in the latest beta. You have to wait until v0.1.1 is released, clone and install `audible-cli` from the master branch or follow the guide in issue #62 and add the plugin command. 

I think I can release v0.1.1 tomorrow, after some final tests and wait for a response from some testers!

**Comment by mkb79** [comment]
@philgoetz audible-cli v0.1.1 is released. This fixes your bug!
